container/docker/factory.go

```diff
@@ -209,15 +209,16 @@ func Register(factory info.MachineInfoFactory, fsInfo fs.FsInfo, ignoreMetrics c
 
 			dockerMetadataDevice, err := dockerutil.DockerMetadataDevice(*dockerInfo)
 			if err != nil {
-				return fmt.Errorf("couldn't determine devicemapper metadata device: %v", err)
+				glog.Errorf("couldn't determine devicemapper metadata device: %v", err)
+				glog.Errorf("devicemapper filesystem stats will not be reported")
```

Comment:
I would only do one Errorf in this case - can you combine this with the previous one?